pom.xml

```diff
@@ -71,7 +71,7 @@
         <git-plugin.version>3.1.0</git-plugin.version>
         <workflow-support-plugin.version>3.3</workflow-support-plugin.version>
         <scm-api-plugin.version>2.2.6</scm-api-plugin.version>
-        <groovy-cps.version>1.30</groovy-cps.version>
+        <groovy-cps.version>1.31</groovy-cps.version>
```

Comment:
You can use a deployed timestamped snapshot (pending JEP-305 configuration for this library).

Reply:
Ok, could you please direct me - probably I have no access to find the right snapshot for groovy-cps, do you know where to look?

Reply:
Pending JEP-305 support for this library (promised by @dwnusbaum at some point), you would have to `mvn deploy` upstream manually, which presumes that you are a plugin developer and thus authenticated to Artifactory.

Reply:
Right, so since groovy-cps#101 is a feature branch and I'm not a plugin developer without access to Artifactory - it will never be available for this pull request. We're stuck here.

Reply:
Ok, now 1.31 release of the groovy-cps is available - and working well with the change.